### PR TITLE
Added flexx recipe

### PR DESCRIPTION
**Pull request by Korijn**
* This recipe is tied to a git revision, so Almar doesn't have to do a release just for us everytime we need a change.
* We append a 4th digit to the version number, so we can use that to pin flexx.

### CODE REVIEW

File: flexx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - tornado
+  run:
+    - python
+    - tornado
```
Comment (Maxyme) [inline flexx/meta.yaml]: tornado min version should be added.

File: flexx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - tornado
```
Comment (Maxyme) [inline flexx/meta.yaml]: Is tornado needed to build?

File: flexx/meta.yaml (hunk 1 above)
Comment (Korijn) [inline flexx/meta.yaml]: Flexx doesn't specify it, so I'll pin it to `>=4.5.1` since that's what we use currently.

File: flexx/meta.yaml (hunk 2 above)
Comment (Korijn) [inline flexx/meta.yaml]: Nope, I'll remove it here.

**Pull request merged**